resources/ecs_task_definitions.go

```diff
@@ -0,0 +1,914 @@
+package resources
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/service/ecs"
+	"github.com/aws/aws-sdk-go-v2/service/ecs/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func EcsTaskDefinitions() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_ecs_task_definitions",
+		Description:  "The details of a task definition which describes the container and volume definitions of an Amazon Elastic Container Service task",
+		Resolver:     fetchEcsTaskDefinitions,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:     "tags",
+				Type:     schema.TypeJSON,
+				Resolver: resolveEcsTask_definitionTags,
```

Comment:
nit: resolver name is not golang convention should be camelcase

Reply:
this fix is in one of my PRs to cq-gen :)